nixos/modules/services/monitoring/alertmanager-executor.nix

```diff
@@ -0,0 +1,70 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.alertmanager-executor;
+  settingsFormat = pkgs.formats.yaml {};
+
```

Comment:
```suggestion
```

Reply:
Fixed in [b8e9eac](https://github.com/NixOS/nixpkgs/pull/189119/commits/b8e9eacbe462ea1aa3ea1c492581c1d529f8c8f3)